Pipfile

```diff
@@ -15,6 +15,7 @@ psycopg2-binary = "~=2.8.6"
 flask-migrate = "~=2.5.3"
 flask-script = "~=2.0.6"
 flask-cors = "~=3.0.9"
+confluent-kafka = "~=1.5.0"
```

Comment:
There is `requests` module required too in the pipfile. Could you add it here?

Reply:
yeah :+1: 
Facing issue while running `python manage.py db upgrade ` - ModuleNotFoundError: No module named 'requests'